src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -1203,22 +1203,30 @@ private void onLink() {
 	}
 
 	private void postCheckLink() {
-		getEntityFinder().configure(CONTAINER, false, new SelectedHandler<Reference>() {
-			@Override
-			public void onSelected(Reference selected) {
-				createLink(selected.getTargetId());
-				getEntityFinder().hide();
-			}
-		});
-		getEntityFinder().show();
+		getEntityFinderBuilder()
+				.setInitialScope(EntityFinderScope.ALL_PROJECTS)
+				.setInitialContainer(EntityFinder.InitialContainer.NONE)
+				.setSelectableTypes(CONTAINER)
+				.setShowVersions(false)
+				.setSelectedHandler((selected, entityFinder) -> {
+					createLink(selected.getTargetId(), entityFinder);
+				})
+				.setTreeOnly(true)
+				.setModalTitle("Create Link to " + entityTypeDisplay)
+				.setHelpMarkdown("Search or Browse to find a Project or Folder that you have access to, and place a symbolic link for easy access")
+				.setPromptCopy("Find a destination and place a link to " + entity.getId())
+				.setSelectedCopy("Destination")
+				.setConfirmButtonCopy("Create Link")
+				.build()
+				.show();

```

Comment:
Most of the changed files are configuring the new entity finder with specified copy